src/main/scala/org/broadinstitute/monster/etl/encode/EncodeEntity.scala

```diff
@@ -0,0 +1,214 @@
+package org.broadinstitute.monster.etl.encode
+
+import enumeratum.{Enum, EnumEntry}
+
+/**
+  * An entity type found in ENCODE that we also want
+  * (with modifications) in our data model.
+  */
+sealed trait EncodeEntity extends EnumEntry {
```

Comment:
I'm not sure we want to repeat this pattern in the pipeline redo. Can we delete it for now? We definitely don't need it for the extraction portion.

Reply:
Alt: We could keep an `EncodeEntity` but delete all the methods / hard-coded collections of strings here. Then we'll just have:
```scala
sealed trait EncodeEntity extends EnumEntry
object EncodeEntity extends Enum[EncodeEntity] {
  case object Biosample extends EncodeEntity
  // etc.
}
```

Reply:
Do we not intend to have fields to keep or rename or anything? Or do we intend to implement that differently moving forward?

Reply:
I'm guessing we'll implement it more like the ClinVar pipeline.